### ISSUE TITLE
abcjs generate midi file error when .abc "T:小星星", 小星星 is Chinese character

### ISSUE THREAD

**Issue by git76607228** [paulrosen/abcjs#182]
hi,
When the .abc file music title set in Chines character, like this file:
```
T: 小星星
M: 4/4
L: 1/4
R: 120
K: C
C C G G | A A G2 | F F E E | D D C2 |
G G F F | E E D2 | G G F F | E E D2 | 
C C G G | A A G2 | F F E E | D D C2 |
```
file encode in UTF8
using this file to generate midi , midi file can not play.
I open midi file using a binary editor, and find an error like this
`00 FF 03 03 5C 0F 66 1F 66 1F`
0x5C0F  0x661F 0x661F is 小星星 unicode.
so this event data length is error.
and I change 0x03 to 0x06 , it works well.
`00 FF 03 06 5C 0F 66 1F 66 1F`
It seems that abcjs consider text length as it''s code length.
and thanks

**Comment by paulrosen** [comment]
Thanks for the bug report. It shouldn't be too long before I can get another version of abcjs released.

**Comment by paulrosen** [comment]
Will be fixed in v5.6.0
